CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Git
+
+This describes the workflow when working with git and the repo.
+
+## Setup
+
+- Fork the repo
+- Download fork
+- `git remote set-url upstream git@github.com:the_repo`
+- run: `tools/install-dev.sh`
+- verify that you are in the right environment (“which python” should be located in local repo)
+
+## Adding a new feature in git
+
+- `git fetch upstream && git reset --hard upstream/master`
+- `git checkout -b my-feature-branch`
+- work:  (may occur many times): 
+  `git add . && git commit -m “my message”`
+- adding: `git fetch upstream`
+  - `git rebase upstream/master`
+  - `git push (git push --set-upstream origin my-feature-branch)`
+- Create PR
+- Review PR yourself
+- Add comments to things that might be non-obvious
+- Add fixes you discovered during self-review
+- Share the PR with the rest of the group.
+- After the PR is approved, use "squash and merge" this hides internal small commits and merges in the master branch.
+
+## General guidelines for PRs
+
+- Try to keep every PR strictly related to a single problem.
+- Try to keep PRs under 500 lines.
+
+## Review guidelines
```

Comment:
I like the examples. They make it very clear for me how to write good code reviews!